### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by rfaircloth-splunk**
## what
* Updates to `Dockerfile`
* Add support for typical conf files

## why
* allow reproducible docker builds

### CODE REVIEW

**Comment by osterman** [comment]
Thanks @rfaircloth-splunk for the contributions! I am away this week, but will work with you to get this merged. We've received an uptick in interest in this project, so looks we should do some housekeeping.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,17 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 RUN apt-get update && \
-    apt-get -y install ruby ruby-dev git make build-essential libicu-dev zlib1g-dev cmake pkg-config
+    apt-get -y install ruby ruby-dev git make build-essential libicu-dev zlib1g-dev cmake pkg-config cmake libssl-dev
+
+# Set the locale
+# make the "en_US.UTF-8" locale so splunk will be utf-8 enabled by default
```
Comment (osterman) [inline Dockerfile]: perhaps we should drop the specific reference to splunk?

File: contrib/syntax.yml [hunk 2]
```diff
@@ -228,4 +228,11 @@ rust:
     open: '/*\n'
     close: ' */\n\n'
     prefix: ' * '
-
+# Conf files i.e. apache config, splunk.conf files
+conf:
+  ext:  ['.conf']
+  after: ['^#!']
```
Comment (osterman) [inline contrib/syntax.yml]: the shebang is probably not relevant for the configs. Probably just make this an empty array (`[]`)

File: README.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Full list of supported arguments:
         -n, --dry-run                    Output the parsed files to STDOUT
         -o, --output-dir DIR             Use DIR as output directory
             --license-file FILE          Use FILE as header (instead of using --license argument)
-            --license [AGPL3|ASL2|BSD-2-CLAUSE|BSD-3-CLAUSE|BSD-4-CLAUSE|GPL3|MIT]
+            --license [AGPL3|APACHE2.0|ASL2|BSD-2-CLAUSE|BSD-3-CLAUSE|BSD-4-CLAUSE|GPL3|MIT]
```
[the remark this answers is not in the document]
Comment (rfaircloth-splunk) [reply README.md]: I've just never used that abbreviation so I missed it the readme and associated additional file changes should be pulled out of the PR

File: lib/copyright_header/parser.rb [hunk 4]
```diff
@@ -228,10 +228,10 @@ def transform(method, path)
               write(path, contents)
             end
           else
-            STDERR.puts "SKIP #{path}; unsupported"
+            STDERR.puts "SKIP #{path}; unsupported #{@syntax.ext(path)}"
           end
         rescue Exception => e
-          STDERR.puts "SKIP #{path}; #{e.message}"
+          STDERR.puts "SKIP EXCEPTION #{path}; #{e.message}"
```
Comment (osterman) [inline lib/copyright_header/parser.rb]: I would prefer: 
```
STDERR.puts "SKIP #{path}; exception=#{e.message}"
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (osterman) [reply README.md]: https://github.com/cloudposse/copyright-header/blob/master/licenses/ASL2.erb

Perhaps ASL2 is not obvious (agree), but then maybe we should consider making an alias instead (though we don't have any existing logic for that)

**Comment by osterman** [comment]
@rfaircloth-splunk just checking in - I believe we're still waiting on some more updates from you.

**Comment by rfaircloth-splunk** [comment]
Yes you are, got caught up in something else, need to get this back to top of heap

File: Dockerfile (hunk 1 above)
Comment (rfaircloth-splunk) [reply Dockerfile]: That's what I get for copy and paste for a similar problem in another docker image I will remove the ref

File: lib/copyright_header/parser.rb (hunk 4 above)
Comment (rfaircloth-splunk) [reply lib/copyright_header/parser.rb]: will do

File: contrib/syntax.yml (hunk 2 above)
Comment (rfaircloth-splunk) [reply contrib/syntax.yml]: Good point

**Comment by rfaircloth-splunk** [comment]
For some reason I could not update this PR making a new one

**Comment by rfaircloth-splunk** [comment]
Figured it out, opened up again

**Comment by rfaircloth-splunk** [comment]
Ping any other concerns not resolved?

**Comment by osterman** [comment]
Thanks @rfaircloth-splunk ! LGTM

**Pull request merged**